amr-wind/equation_systems/icns/icns_advection.cpp

```diff
@@ -109,6 +109,24 @@ void MacProjOp::init_projector(const amrex::Real beta) noexcept
     m_need_init = false;
 }
 
+void MacProjOp::set_inflow_velocity(amrex::Real time)
+{
+    auto& velocity = m_repo.get_field("velocity");
+    auto& u_mac = m_repo.get_field("u_mac");
+    auto& v_mac = m_repo.get_field("v_mac");
+    auto& w_mac = m_repo.get_field("w_mac");
+
+    for (int lev = 0; lev < m_repo.num_active_levels(); ++lev) {
+        amrex::Array<amrex::MultiFab*, ICNS::ndim> mac_vec;
```

Comment:
Condense all this:
`amrex::Array<amrex::MultiFab*, AMREX_SPACEDIM> mac_vec = {AMREX_D_DECL(&u_mac(lev), &w_mac(lev), &w_mac(lev))};`